packages/admin/admin-rte/src/core/Controls/LinkControls.tsx

```diff
@@ -52,6 +63,10 @@ export default (p: IControlProps) => {
 };
 
 declare module "@mui/material/styles" {
+    interface ComponentsPropsList {
+        CometAdminRteLinkControls: RteLinkControlsProps;
```

Comment:
@jamesricky shouldn't this be optional?

Reply:
@johnnyomair No, only in the `Components` interface.

@juliawegmayr when adding the this to the `ComponentsPropsList` interface, please also add that type to the `Components` interface as `defaultProps`.